cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt

```diff
@@ -23,21 +28,33 @@ private val os by lazy {
 private enum class OS { MACOS, WINDOWS, LINUX }
 
 private object DebugPortAlloc {
-    private var basePort = 5005
+    private var basePort = base_debug_port
     internal fun next() = basePort++
 }
 
 private object MonitoringPortAlloc {
-    private var basePort = 7005
+    private var basePort = base_monitoring_port
     internal fun next() = basePort++
 }
 
 fun main(args: Array<String>) {
     val startedProcesses = mutableListOf<Process>()
     val headless = ((!isTmux() && GraphicsEnvironment.isHeadless()) || args.contains(HEADLESS_FLAG))
     val capsuleDebugMode = args.contains(CAPSULE_DEBUG_FLAG)
+    try{
+        base_debug_port = Integer.valueOf(args.first { it -> it.contains(BASE_DEBUG_PORT_FLAG)}.removePrefix(BASE_DEBUG_PORT_FLAG))
+        println("base_debug_port set to: $base_debug_port")
+    }catch(e: Throwable){
```

Comment:
This should only be catching `NumberFormatException` because anything else is likely to be too serious for us to continue.